.travis/wait_for_services.rb

```diff
@@ -40,6 +40,6 @@
     end
   end
 end
-puts 'Waiting 2 minutes for agent to complete it\'s first round...'
-sleep 120
+puts 'Waiting 1 minute for agent to complete it\'s first round...'
+sleep 60
```

Comment:
Not really sure, 1 minute should be more than enough on these tests.
I guess there is not a really way to know when the agent has finished (except for reading that every resource is in place, which is what the tests does).
Anyway most of the tests have a wait mechanism to not fail immediately but give some time before actually failing.